builder/dao.go

```diff
@@ -150,12 +150,18 @@ func assembleExpression(field, op string) string {
 	return quoteField(field) + op + "?"
 }
 
-func orderBy(field, order string) (string, error) {
-	realOrder := strings.ToUpper(order)
-	if realOrder != "ASC" && realOrder != "DESC" {
-		return "", errOrderByParam
+func orderBy(orderMap []eleOrderBy) (string, error) {
+	var orders []string
+	for _, orderInfo := range orderMap {
+		realOrder := strings.ToUpper(orderInfo.order)
+		if realOrder != "ASC" && realOrder != "DESC" {
+			return "", errOrderByParam
+		}
+		order := fmt.Sprintf("%s %s", quoteField(orderInfo.field), realOrder)
+		orders = append(orders, order)
 	}
-	return fmt.Sprintf("%s %s", quoteField(field), realOrder), nil
+	orderby := strings.Join(orders, ", ")
```

Comment:
It'll be better to remove the blank space after the comma. People may be puzzled when adding test cases but unaware of the existing of the blank space.

Reply:
The purpose of spaces is to make SQL look pretty, but It's better to make code clean, fixed that.